string actionName
string[] orderedArgs
---
ActionPDDLBinding pddlBindings
